aziotctl/aziotctl-common/src/system_logs.rs

```diff
@@ -0,0 +1,24 @@
+use std::ffi::OsString;
+use std::process::Command;
+
+use anyhow::{Context, Result};
+
+#[allow(clippy::missing_errors_doc)]
+pub fn get_system_logs(processes: &[&str], additional_args: &[OsString]) -> Result<()> {
```

Comment:
`additional_args: &[&OsStr]`

For the caller, `options.args.iter().map(AsRef::as_ref)`